doc/languages.rst

```diff
@@ -198,6 +198,35 @@ to view the docstring of the syntax checker.  Likewise, you may use
          The cppcheck suppressions file to use via the
          ``--suppressions-list=`` argument.
 
+.. supported-language:: CUDA C/C++
+   :index_as: CUDA
+
+   .. syntax-checker:: cuda-nvcc
+
+      Checks syntax for CUDA C/C++ using the nvcc compiler bundled in the
```

Comment:
Please turn `nvcc` into a link to its webpage (the same URL as the one you used for the checker definition).

```suggestion
      Checks syntax for CUDA C/C++ using the `nvcc <https://developer.nvidia.com/cuda-llvm-compiler>` compiler bundled in the
```